pkg/util/log/middleware.go

```diff
@@ -2,22 +2,41 @@ package log
 
 import (
 	"bufio"
+	"context"
 	"fmt"
 	"net"
 	"net/http"
 	"strings"
 	"time"
 
 	kitlog "github.com/go-kit/kit/log"
+	uuid "github.com/satori/go.uuid"
 )
 
+const (
+	KubernikusRequestID = "Kubernikus-Request-Id"
```

Comment:
Quoting the golang docs for `WithValue`:

> The provided key must be comparable and should not be of type string or any other built-in type to avoid collisions between packages using context. Users of WithValue should define their own types for keys. ...

Reply:
The original blog post about `Context` gives an example on how they suggest this shut be used: https://blog.golang.org/context#TOC_3.2.

Reply:
Thanks, good tip. Much better...